### PR TITLE
Add Download File Preview functionality

### PR DESCRIPTION
**Pull request by Trukes**
The objective of this PR is to add the Download File Preview code with tests.

It's related with this issue: https://github.com/crowdin/crowdin-api-client-php/issues/129

### CODE REVIEW

**Review by andrii-bodnar — APPROVED** [review]
@Trukes thank you!

**Comment by andrii-bodnar** [comment]
@Trukes some checks have failed, please take a look

File: .phpunit.result.cache [hunk 1]
```diff
@@ -0,0 +1 @@
+{"version":1,"defects":{"CrowdinApiClient\\Tests\\Api\\FileApiTest::testDelete":4},"times":{"CrowdinApiClient\\Tests\\Api\\FileApiTest::testList":0.078,"CrowdinApiClient\\Tests\\Api\\FileApiTest::testGetAndEdit":0.001,"CrowdinApiClient\\Tests\\Api\\FileApiTest::testUpdate":0.001,"CrowdinApiClient\\Tests\\Api\\FileApiTest::testRestore":0,"CrowdinApiClient\\Tests\\Api\\FileApiTest::testCreate":0,"CrowdinApiClient\\Tests\\Api\\FileApiTest::testDelete":0.003,"CrowdinApiClient\\Tests\\Api\\FileApiTest::testDownloadPreview":0.001,"CrowdinApiClient\\Tests\\Api\\FileApiTest::testDownload":0.001,"CrowdinApiClient\\Tests\\Api\\FileApiTest::testListRevisions":0.001,"CrowdinApiClient\\Tests\\Api\\FileApiTest::testGetRevision":0}}
```
Comment (andrii-bodnar) [inline .phpunit.result.cache:1]: please remove this file

Comment (Trukes) [reply .phpunit.result.cache]: Done 👍🏻

**Pull request merged**